README.md

```diff
@@ -1 +1,95 @@
-# awesome-book-ES6
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
- [ ] Kindly remove the Microverse logo from this file. If you have your customized logo, you can use it, but if you don't, kindly remove it. 👍